extension/src/test/fixtures/expShow/dataTypes.ts

```diff
@@ -0,0 +1,193 @@
+import { ExperimentsOutput } from '../../../cli/reader'
+import {
+  Column,
+  ColumnType,
+  Row,
+  TableData
+} from '../../../experiments/webview/contract'
+
+export const dataTypesOutput: ExperimentsOutput = {
```

Comment:
We'll probably add more types here, I just included what has been asked for and we can work from there- I haven't tried nested lists or objects, and `null` breaks the table when given but it doesn't occur in JSON.